python/pyrightconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    "reportMissingImports": false,
+    "include": [
+        "semantic_kernel/**/*.py"
+    ],
+    "pythonVersion": "3.10",
+    "exclude": [
+        "semantic_kernel/memory/*",
```

Comment:
Are we moving to pyright now?

Reply:
not fully yet, because there is a ton of work involved since it is much stricter, but I do plan on extending that overtime and at some point switching over, might be worth a spike...